src/MSL/math.c

```diff
@@ -0,0 +1,166 @@
+#include "./math.h"
```

Comment:
When you have the chance, fix this include to be the explicit path: "MSL/math.h"

Reply:
done https://github.com/doldecomp/melee/pull/235/commits/ed6194cefb838f1ef1b3ca7e05e1973bcdfce662